app/scripts/lib/channels/inter-tab.js

```diff
@@ -8,23 +8,60 @@
  */
 
 define([
-  'crosstab'
-], function (crosstab) {
+  'backbone',
+  'crosstab',
+  'underscore'
+], function (Backbone, crosstab, _) {
   'use strict';
 
-  function InterTabChannel(options) {
+  function BroadcastChannelAdapter(options) {
     options = options || {};
-    this._crosstab = options.crosstab || crosstab;
+    var win = options.window || window;
+
+    this._broadcastChannel = new win.BroadcastChannel('firefox_accounts');
+    this._broadcastChannel.onmessage = this.onMessage.bind(this);
   }
 
-  InterTabChannel.prototype = {
+  BroadcastChannelAdapter.prototype = {
```

Comment:
Add documentation on each adapter and why BroadcastChannel is better than crosstab.